Mark "Frauentag" as a public holiday for Berlin

MaherSaleem opened the pull request:
After 2019, Frauentag is considered as a public holiday in Berlin

commenthol commented:
Hi @MaherSaleem,
8th of March is already a public holiday in Berlin. What do you intent with your change?

MaherSaleem commented:
> Hi @MaherSaleem, 8th of March is already a public holiday in Berlin. What do you intent with your change?

Hi @commenthol, I just added a name for name (since now it's just called "03-08". Moreover, I've added `type: public` since it's a public holiday

arkraft commented:
Hi,

the day is already a public holiday:
<img width="706" alt="image" src="https://github.com/commenthol/date-holidays/assets/7426376/23aa73e7-1a98-40a5-9795-b7039e7f97b4">

Also, _name is translated in a different file. The name is not "03-08", it is mapped with this in the file [data/names.yml](https://github.com/commenthol/date-holidays/blob/master/data/names.yaml#L112-L133)
<img width="351" alt="image" src="https://github.com/commenthol/date-holidays/assets/7426376/02bcaf18-2c18-4d62-a6c5-c02e8aac3f84">
